moviepy/video/VideoClip.py

```diff
@@ -116,6 +117,33 @@ def h(self):
     def aspect_ratio(self):
         return self.w / float(self.h)
 
+    def __copy__(self):
+        """Mixed copy of the clip.
+
+        Returns a shallow copy of the clip whose mask and audio will
+        be shallow copies of the clip's mask and audio if they exist.
+
+        This method is intensively used to produce new clips every time
+        there is an outplace transformation of the clip (clip.resize,
+        clip.subclip, etc.)
+
+        Acts like a deepcopy except for the fact that readers and other
+        possible unpickleables objects are not copied.
+        """
+        cls = self.__class__
+        new_clip = cls.__new__(cls)
+        for attr in self.__dict__:
+            value = getattr(self, attr)
+            if attr in ("mask", "audio"):
```

Comment:
I only just noticed this, but aren't lines 137-138 and 141-142 doing the same thing?

Reply:
Yes, I didn't notice :man_facepalming:  Let me fix it

Reply:
Fixed in https://github.com/Zulko/moviepy/commit/fac722df7c4c0c33a9d98877b13647f4115f7b53